### PR TITLE
fix(progress-bar-v2): use global state countdown

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Use global rather than local state for the countdown

# To Test

1. Place order
2. While the countdown is running, open the activity modal
* Countdown should continue from where it was rather than start at 15s

### CODE REVIEW

**Comment by elena-zh** [comment]
@alfetopito , the build has failed. Could you please retrigger it again?

**Review by anxolin — COMMENTED** [review]
👍

**Review by elena-zh — COMMENTED** [review]
Hey @alfetopito , thanks for addressing this!
But 2 issues: countdown animation does not look good: 
1. each time when I open Account modal, the circle starts progress from the top 
image
2.Countdown is running to fast and starts again and again from the top from 7sec. See the video: https://www.loom.com/share/7aabf377939e4977a4a282f1b0637c20

**Comment by alfetopito** [comment]
> Hey @alfetopito , thanks for addressing this! But 2 issues: countdown animation does not look good:
> 
>     1. each time when I open Account modal, the circle starts progress from the top
>        image
>        2.Countdown is running to fast and starts again and again from the top from 7sec. See the video: https://www.loom.com/share/7aabf377939e4977a4a282f1b0637c20

Yes, I know.
I forgot to mention that in the description.

That's a good to have which maybe @fairlighteth can help with if he has the time.

File: apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/styled.ts [hunk 1]
```diff
@@ -318,12 +318,13 @@ export const CircularProgress = styled.svg`
...
 
-export const CircleProgress = styled.circle`
+export const CircleProgress = styled.circle<{ duration: number }>`
   fill: none;
   stroke: #012f7a;
   stroke-width: 7;
   stroke-linecap: round;
-  animation: ${progressAnimation} 15s linear infinite;
+  // TODO: start animation at different position based on how far from 15s it is
+  animation: ${progressAnimation} ${({ duration }) => duration}s linear infinite;
```
[the remark this answers is not in the document]
Comment (alfetopito) [reply apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/styled.ts]: Never mind, I _think_ I managed it 🤞

**Review by elena-zh — APPROVED** [review]
Works, thank you!

**Pull request merged**